node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -497,13 +512,58 @@ pub fn get_block(hash: Result<(Hash,), jsonrpc_core::Error>) -> JsonRpcResultAsy
             .send(GetItemBlock { hash })
             .then(move |res| match res {
                 Ok(Ok(output)) => {
-                    let value = match serde_json::to_value(output) {
+                    let vtt_hashes: Vec<_> = output
+                        .txns
+                        .value_transfer_txns
+                        .iter()
+                        .map(|txn| txn.hash())
+                        .collect();
+                    let drt_hashes: Vec<_> = output
+                        .txns
+                        .data_request_txns
+                        .iter()
+                        .map(|txn| txn.hash())
+                        .collect();
+                    let ct_hashes: Vec<_> = output
+                        .txns
+                        .commit_txns
+                        .iter()
+                        .map(|txn| txn.hash())
+                        .collect();
+                    let rt_hashes: Vec<_> = output
+                        .txns
+                        .reveal_txns
+                        .iter()
+                        .map(|txn| txn.hash())
+                        .collect();
+                    let tt_hashes: Vec<_> = output
+                        .txns
+                        .tally_txns
+                        .iter()
+                        .map(|txn| txn.hash())
+                        .collect();
+
+                    let txns_hashes = serde_json::json!({
+                        "txns_hashes" : {
+                            "mint" : output.txns.mint.hash(),
```

Comment:
The mint hash is available as `block["block_header"]["merkle_roots"]["mint_hash"]`, but let's also keep it here for consistency.

Reply:
Yeah, I know, but as you mentioned I specifically added it here too for consistency.